source/OutfitterPanel.cpp

```diff
@@ -764,7 +764,7 @@ void OutfitterPanel::FailSell(bool toStorage) const
 									"because that would cause your ship's \"" + it.first +
 									"\" value to be reduced to less than zero. "
 									"To " + verb + " this outfit, you must " + verb + " the " +
-									sit.first->Name() + " outfit first."));
+									sit.first->DisplayName() + " outfit first."));
```

Comment:
Another opportunity for confusion. _Which "flux capacitor" do I need to uninstall? I have two different ones!_

Reply:
They will likely have different stats. This is actually the feature that content creators are looking for.